src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow
```

Comment:
This is not how Flow works I'm afraid. It will "pass" even without annotations at all—because it doesn't know anything. So the key takeaway is: we either add annotations to every function in the file, or we don't do it at all.